bindata/network/ovn-kubernetes/common/002-rbac-node.yaml

```diff
@@ -184,7 +185,11 @@ rules:
   - get
   - list
   - watch
-
+{{- if .IsNetworkTypeMigration }}
+- apiGroups: ['network.openshift.io']
+  resources: ['hostsubnets']
+  verbs: ['get', 'list', 'create', 'patch']
```

Comment:
Do we need create and patch for hostsubnets in ovnkube-node?

Reply:
Yes, when rolling back, we need to create hostsubnet CRs in ovnkube-node.